build/params_mainnet.go

```diff
@@ -81,3 +81,6 @@ const PropagationDelaySecs = uint64(6)
 
 // BootstrapPeerThreshold is the minimum number peers we need to track for a sync worker to start
 const BootstrapPeerThreshold = 4
+
+// we skip checks on message validity in this block to sidestep the zero-bls signature
+var WhitelistedBlock = MustParseCid("bafy2bzaceapyg2uyzk7vueh3xccxkuwbz3nxewjyguoxvhx77malc2lzn2ybi")
```

Comment:
Can we have a set of `WhitelistedBlocks` and maybe a bitfield/object describing _what_ checks are skipped? E.g.,

```
type SkippedChecks {
...
}
var WhitelistedBlocks map[cid.Cid]SkippedChecks
```

Reply:
I don't think we plan to have more than one of those blocks ever

Reply:
Of course not.